README.md

```diff
@@ -52,17 +52,17 @@ under the *Web* tab in QGIS. It opens a new window, which lets the user control
   exported on their local disk. A user can also add their personal MapBox API key if they wish to add MapBox basemaps to
   their project. In this tab a user can also define the logging level mainly for development purpose.
 
-From the *About* tab a user can see the basic infomation about the version they are using and find relevant links. 
+From the *About* tab a user can see the basic infomation about the version they are using and find relevant links.
 
 ### Supported styling and layer types
 
 Currently the plugin supports exporting line, point and polygon geometries. The cartographic capabilities in QGIS are vast and can become very complex, and currently the plugin supports only basic styles.
```

Comment:
Describe  what happens if a map contains unsupported layers ? It will still export but only contain the layers that were supported by the conversion process?

Reply:
I've also captured "defensive strategies" in https://foursquare.atlassian.net/browse/UN-5366, there's definitely room for improvement on the UX side.